Source/Core/VideoBackends/OGL/SamplerCache.cpp

```diff
@@ -134,8 +132,26 @@ void SamplerCache::SetParameters(GLuint sampler_id, const Params& params)
 	if (GLInterface->GetMode() == GLInterfaceMode::MODE_OPENGL)
 		glSamplerParameterf(sampler_id, GL_TEXTURE_LOD_BIAS, (s32)tm0.lod_bias / 32.f);
 
-	if (g_ActiveConfig.iMaxAnisotropy > 0 && g_ogl_config.bSupportsAniso)
+	GLint min_filter = min_filters[tm0.min_filter];
+	GLint mag_filter = tm0.mag_filter ? GL_LINEAR : GL_NEAREST;
+
+	if (g_ActiveConfig.iMaxAnisotropy > 0 && g_ogl_config.bSupportsAniso &&
+	    !IsBpTexMode0PointFiltering(tm0))
+	{
+		// https://www.opengl.org/registry/specs/EXT/texture_filter_anisotropic.txt
+		// For predictable results on all hardware/drivers, only use one of:
+		//	GL_LINEAR + GL_LINEAR (No Mipmaps [Bilinear])
+		//	GL_LINEAR + GL_LINEAR_MIPMAP_LINEAR (w/ Mipmaps [Trilinear])
+		// Letting the game set other combinations will have varying arbitrary results;
+		// possibly being interpreted as equal to bilinear/trilinear, implicitly
+		// disabling anisotropy, or changing the anisotropic algorithm employed.
+		min_filter = (tm0.min_filter & 3) == TexMode0::TEXF_NONE ? GL_LINEAR : GL_LINEAR_MIPMAP_LINEAR;
```

Comment:
@lioncash This test expression is pulled from the D3D backend ([1](https://github.com/dolphin-emu/dolphin/blob/54c2800bb4a7a5bd7ead7319a28da9269a3f8ed3/Source/Core/VideoBackends/D3D/D3DState.cpp#L260), [2](https://github.com/dolphin-emu/dolphin/blob/54c2800bb4a7a5bd7ead7319a28da9269a3f8ed3/Source/Core/VideoBackends/D3D/D3DState.cpp#L313)) and is identical across all four backends (including the [software backend](https://github.com/dolphin-emu/dolphin/blob/54c2800bb4a7a5bd7ead7319a28da9269a3f8ed3/Source/Core/VideoBackends/Software/TextureSampler.cpp#L72)). Do you want me to refactor and pull it out into a function as well?